.changeset/quiet-pens-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+'@talend/react-dataviz': patch
+---
+
+fix(dataviz): bundle typing in the package
```

Comment:
```suggestion
fix: bundle typing in the package that produce the following error:

```
Could not find a declaration file for module '@talend/react-dataviz' since the 3.0
```
```